tempest/api/workloadmgr/workload_policy/test_workload_policy.py

```diff
@@ -597,7 +597,7 @@ def test_7_policywith_scheduler_retension(self):
                 LOG.debug("Command workload create executed correctly")
 
             time.sleep(10)
-            self.workload_id = query_data.get_workload_id(
+            self.workload_id = query_data.get_workload_id_in_creation(
```

Comment:
At line number - 669 - We have a code - self.workload_id2 = query_data.get_workload_id(...) - Why it is not replaced to get_workload_id_in_creation() ?

Reply:
Fixing tests which are failing during execution. We can change it for all tests as well. Will check with savitha, if we need to do that?

Reply:
One more point - While looking for get_workload_id() in tempest code branch. I can see it is also available in "restore/test_volume_vol.py" and "restore/test_volume_booted.py" file. 
Any specific reason why we have not changed get_workload_id() to get_workload_id_in_creation() in these 2 files ?

Reply:
> Fixing tests which are failing during execution. We can change it for all tests as well. Will check with savitha, if we need to do that?

Yes, it would be great if we cross check the remaining positions where get_workload_id() is called AND if it really require a change to get_workload_id_in_creation().

Reply:
done all the changes.